react/components/SKUSelector/components/SelectorItem.js

```diff
@@ -16,8 +16,14 @@ class SelectorItem extends PureComponent {
 
   render() {
     return (
-      <div className={`${VTEXClasses.SELECTOR__ITEM} di ba bw1 pointer flex items-center ${this.props.isSelected ? 'b--blue' : 'b--transparent'}`} onClick={this.handleClick}>
```

Comment:
As I said, you reverted the previous changes.

`flex items-center ` and `b--blue`

Reply:
Done!